tests/compile-fail/never_transmute_humans.rs

```diff
@@ -1,17 +0,0 @@
-// This should fail even without validation
```

Comment:
This causes an ICE

Reply:
That's a bug, isn't it? Why remove the test?

Reply:
it causes an ICE in rustc during compilation, not miri: http://play.integer32.com/?gist=76ba0e33563d046120e629846294f4a8&version=nightly&mode=debug

Reply:
still... ignoring it for now is better so we can add it once upstream fixes it

Reply:
Ah, I see.

Is this reported upstream? The ignore-test comment should link to the issue.

Reply:
rust-lang/rust#50570